recipes/brewer2mpl/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.4.1" %}
+
+package:
+  name: brewer2mpl
+  version: {{ version }}
+
+source:
+  fn: brewer2mpl-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/fa/fd/838695792e82978a708fc5ca6ad5d085923d8d76f5e01875078e837827cf/brewer2mpl-{{ version }}.tar.gz
```

Comment:
Change the long URL to `https://pypi.io/packages/source/b/brewer2mpl/brewer2mpl-{{ version }}.tar.gz`